src/changelog/unreleased.md

```diff
@@ -12,6 +12,7 @@ on how to add them:
 ### Added
 
 - Add `Window::turbo()`, implemented on X11, Wayland, and Web.
+- Implement `ActiveEventLoopExtWayland` and `ActiveEventLoopExtX11` for `EventLoop`.
```

Comment:
This is not correct anymore. It should mention the new traits.

Reply:
Updated